crs_reference.md

```diff
@@ -0,0 +1,26 @@
+This document explains relationship between all the Custom Resources (CRs)
+required to create the target Kubernetes cluster on bare metal machine and how
+they reference to each other.
+
+## Environment Variables
+
+The user is required to set the following environment variables before applying
```

Comment:
can you please explain that which CRs are you talking about and from which repository .. is it from the operators or metal3 dev env e.t.c.